core/test/functional/api/v2/admin/utils.js

```diff
@@ -45,9 +66,25 @@ const expectedProperties = {
                 'secret',
                 'integration_id'
             )
-            .value()
 };
 
+_.each(expectedProperties, (value, key) => {
+    if (!value.__wrapped__) {
+        return;
+    }
+
+    /**
+     * @deprecated: x_by
+     */
+    expectedProperties[key] = value
+        .without(
```

Comment:
This is a fine approach as if follows the convention of blacklisting fields as done for every resource in the module. Just made me think about how this still exposes us to unintentionally returning data when adding new tables/fields to the schema. 

The approach that would be a bit more secure - having whitelisted fields that we intend to expose. That would require a kind of API description that we don't have at the moment. 

Just a thought no need to change anything here :smiley:

Reply:
> having whitelisted fields that we intend to expose

yeah that would be better 👍The API response props are currently pulled from the db schema. We shouldn't do that anymore now that we have multiple API versions. We should change that in the future. Maybe if we add a another API version.